src/util.h

```diff
@@ -270,6 +270,19 @@ std::string VecToStr(const std::vector<T>& vIn)
     return s.substr(0, s.length()-1).c_str();
 }
 
+template <typename T>
+int FindIndexOf(const std::vector<T>& vIn, const T& entry)
+{
+    if (vIn.empty())
```

Comment:
do we need this check?
`std::find` returns `.end()`for empty vector.

Reply:
Done